packages/chain-adapters/src/evm/types.ts

```diff
@@ -6,7 +6,13 @@ import type * as common from '../types'
 
 export type Account = {
   nonce: number
-  tokens?: common.AssetBalance[]
+  tokens?: Token[]
+}
+
+export type Token = common.AssetBalance & {
+  symbol: string
+  name: string
+  precision: number
 }
```

Comment:
can use the type `MinimalAsset` from asset service

```suggestion
  tokens?: Token[]
}

export type Token = common.AssetBalance & MinimalAsset
```

Reply:
would it worth commenting that the additional metadata is used for arb nova placeholder assets?

Reply:
I can add a quick comment regarding the metadata, but imo I think separation between chain-adapters and asset service makes sense and would rather not start mixing types there.